app/src/main/res/values/styles.xml

```diff
@@ -6,4 +6,5 @@
         <item name="postSplashScreenTheme">@style/Theme.Notificity</item>
         <item name="windowSplashScreenAnimationDuration">15000</item>
     </style>
+
```

Comment:
@mrwhoknows55 this is the only change in the file. It would be out of this PR if not for this change

Reply:
ide added empty line, will remove rollback it